src/sinks/kafka.rs

```diff
@@ -150,6 +151,13 @@ impl KafkaSinkConfig {
 
         self.auth.apply(&mut client_config)?;
 
+        if let Some(statistics_interval_ms) = &self.statistics_interval_ms {
+            client_config.set(
+                "statistics.interval.ms",
+                &statistics_interval_ms.to_string(),
+            );
+        }
```

Comment:
Does this mean no metrics by default? I'm honestly not sure that we need to make this configurable. We could just set it ourselves to every second or two.

Reply:
Hm, yes a default for this sounds like a good idea. Also agree that configuring the specific interval sounds like an edge case.

If I remove this option the interval would still be configurable by the `librdkafka_options` field in the config.

I feel that having an explicit option would make the existence of the statistics more discoverable, but maybe this is not needed?

Reply:
> If I remove this option the interval would still be configurable by the `librdkafka_options` field in the config.

That seems fine to me.

> I feel that having an explicit option would make the existence of the statistics more discoverable, but maybe this is not needed?

Yeah, I think that these metrics should be discovered in the same way as any other metrics that we emit from components.